### ISSUE TITLE
Misleading "Unknown repository type" error message when attempting to add conan remote URL from packagedependencies.txt

### ISSUE THREAD

**Issue by jim-bcom** [b-com-software-basis/remaken#21]
**HOW TO REPRODUCE**
* Create a `packagedependencies.txt` file with a conan dependency using the remaken feature that allows to specify the URL of a conan remote, for instance:
```
ceres-solver|2.1.0|ceres-solver|conancenter@conan|conancenter#https://center.conan.io#0
```
* Ensure the referenced remote alias (here `conancenter`) exists with a **different** URL (can be valid or not)
```
> conan remote list
conancenter: https://center.conan.io.foo [Verify SSL: True]
```
* run `remaken install` on that file

**EXPECTED BEHAVIOR**
* dependency gets installed using either the already existing remote URL or the one specified in `packagedependencies.txt` if the policy is for it to override the existing environnment. It could also print a warning message to notify the user that there was a conflict and how it was resolved (i.e. which URL has been picked).
OR
* an error occurs because of the conflict and asks the user to resolve it
Anyway, something explicit for the user.

**OBSERVED BEHAVIOR**
* An unclear error message gets printed
```
> remaken install
REMAKEN_PKG_ROOT environment variable exists
REMAKEN_PKG_ROOT = D:/users/username/.remaken/package
=> Installing conan::ceres-solver/2.1.0@|win-cl-14.1|x86_64|17|shared|release|
Adding conan remote conancenter https://center.conan.io at 0
[2023-06-07 15:32:51.928325] [0x00028420] [error]   Unknown repository type
```

**NOTES**
* This seems to only happen when the remote alias exists, but with a different URL. If the URL is the same or if the remote alias does not exist, the package gets installed without the `Adding conan remote conancenter https://center.conan.io at 0` message.
* This seems to happen only on Windows and not on Linux (TBC?)
* This message has already been reported in an other issue, that has been marked as fixed: https://github.com/b-com-software-basis/remaken/issues/8

OS: Windows 10
remaken: 1.9.4

**Comment by stefled** [comment]
Issue reproduced

**Solution : Clarify check conditions and messages.**

New information messages are displayed when the remote name or remote url already exists.
In this case, adding remote is not execute.

message when remote name exists : _conan remote named %remote_name% already exists - Do nothing_
message when remote url exists : _conan remote url%remote_url% already exists - Do nothing_
message when name and url don't exist : 
_conan remote named %remote_name% and url %remote_url% were not found!
Adding conan remote %remote_name% %remote_url% at %remote_index%_

fix in next version 1.10.0

**Comment by jim-bcom** [comment]
Confirmed with remaken 1.10.0.

New output:
```
=> Installing conan::ceres-solver/2.1.0@|linux-gcc|x86_64|17|shared|release|
conan remote named conancenter already exists - Do nothing
[...]
ceres-solver/2.1.0: Not found in local cache, looking in remotes...
ceres-solver/2.1.0: Trying with 'conan-solar'...
ceres-solver/2.1.0: Trying with 'conancenter'...
ERROR: HTTPSConnectionPool(host='center.conan.foo', port=443): Max retries exceeded with url: /v1/ping (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0725de6d40>: Failed to establish a new connection: [Errno -2] Name or service not known'))

Unable to connect to conancenter=https://center.conan.foo
1. Make sure the remote is reachable or,
2. Disable it by using conan remote disable,
Then try again.
[2023-12-08 14:22:47.899604] [0x00007f17b9c7bd80] [error]   Unknown repository type
```

Message adds an information about what happens with conan remote overriding.

But in this particular case where the URL is wrong, there's still this message I find misleading of `Unknown repository type`.

In this case the error reporting is mainly handled by conan itself.

I don't know if it's possible to get a more correct error message from remaken such as, I don't know, "Error with conan dependency '...' ", or "Failed to install dependencies, check logs", or something else...

But in this case, maybe it's just best and simpler to just print nothing ?

It's measleading because, according to the [documentation](https://github.com/b-com-software-basis/remaken#packagedependenciesextra-packages-file-syntax), repository type here is `conan`, and it's not unknown.

**Comment by stefled** [comment]
The fix on bug #25 should normally improve error text displayed.

Remaken only adds conan remote if remote name and remote url doesn't already exist.
it's seems you have already a conan remote named `conancenter` then it is not updated.
your `conancenter` remote has an invalid url then it can't be reachable.
